react/src/lib/components/DeckGLMap/layers/terrain/map3DLayer.ts

```diff
@@ -404,6 +404,23 @@ export default class Map3DLayer extends CompositeLayer<
         );
         return [layer];
     }
+
+    getLegendData(): ContinuousLegendDataType {
+        const min = this.props.colorMapRange
```

Comment:
Should also check if `propertyTexture` is defined here.

Reply:
> @hkfb but propertyTexture is an string . How will I check that for range ? You mean to show legend only when propertyTexture is defined ?

If `propertyTexture` is not defined or empty, then the `propertyValueRange` is not used for color sampling. The `meshValueRange` is used instead.